[Rewrite] Function Wikis - #1 (#201)
NilPointer-Software/bdfd-wiki: src/bdscript/username.md

Returns a user's username.

## Usages
There are two usages of this function :

### Usage #1
#### Syntax
```
$username
```
Returns the username of the user who triggered the command.

#### Example
```
$nomention
Hello $username!
```
![example1](https://user-images.githubusercontent.com/69215413/114783581-fc88e180-9d47-11eb-91a6-02e60b20fcf4.png)

### Usage #2
#### Syntax
```
$username[User ID]
```

> Returns username of a specific user using it's ID.

Reviewer: ```suggestion
Returns the username of a specific user given its ID.
```

##### Parameters

> - `User ID` : The ID of the user whose username is to get.

Reviewer: ```suggestion
- `User ID` : The user to get the username for.
```
